internal/pass/pairing/pairing.go

```diff
@@ -148,11 +144,20 @@ func (p *Pairing) sendTokenAndCloseConn(extID, deviceID string, conn *websocket.
 	if err != nil {
 		return fmt.Errorf("Failed to generate ext proxy token: %v", err)
 	}
+	var syncToken string
+	if pairingInfo.Device.FCMToken != "" {
+		syncToken, err = p.signSvc.SignAndEncode(sign.Message{
+			ConnectionID:   pairingInfo.Device.FCMToken,
+			ExpiresAt:      time.Now().AddDate(1, 0, 0),
```

Comment:
can you put it into const or pass via env?

Reply:
done